src/gui/util_gui.cpp

```diff
@@ -343,6 +345,22 @@ namespace Util {
 		}
 		return maybe_markup;
 	}
+	
+	void Marker::drawCenterMarker(QPainter* painter, const QPointF& center)
```

Comment:
Which style shall we eventuall adopt to implement namespaced free functions: `::` style as here, or explicit `namespace` blocks as elsewhere in this file?

Reply:
I didn't want to insist on a change here. It is just that I stumbled about the free function looking like class member, different from the style used for other free functions in this particular file.

Reply:
The file is [already inconsistent](https://github.com/OpenOrienteering/mapper/blob/9adf5c7053de8dafeb03219815d1c89efe40eba4/src/gui/util_gui.cpp#L217-L232) and needs to be checked for style anyway.

I've introduced `namespace` in my change but my personal taste is towards use of scope resolution operator (`::`) in function definitions. Supporting arguments are:
- Creator uses this notation when triggering the "Add function definition to .cpp file". If we use enclosing name spaces, programmers can forget about this editor function or have to rewrite the generated code anyway.
- The accompanying name space block enforces code indentation which is limiting usable horizontal space. We have given up on indentation within global `OpenOrienteering` name space but not the deeper nested ones.

Whichever approach we take, existing code base still needs adjustments.